components/Button.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
```

Comment:
t think it is not necessary to import React, just importing useState should be enough.